.expeditor/config.yml

```diff
@@ -81,183 +81,183 @@ subscriptions:
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:knife-windows-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:mixlib-install-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:mixlib-versioning-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:win32-event-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:win32-eventlog-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:win32-ipc-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:win32-mmap-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:win32-mutex-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:wmi-lite-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-azurerm-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-ec2-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-digitalocean-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-dokken-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-google-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-hyperv-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-inspec-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-vagrant-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:kitchen-vcenter-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:test-kitchen-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:foodcritic-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:chefspec-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:fauxhai-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:berkshelf-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:dco-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:chef-sugar-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:opscode-pushy-client-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:nokogiri-*
-    actions:
-      - bash:.expeditor/update_dep.sh
-  - workload: ruby_gem_published:stove-*
-    actions:
-      - bash:.expeditor/update_dep.sh
+  # - workload: artifact_published:stable:chef:14*
```

Comment:
These got commented out because they are not working right now, correct?

Reply:
yep they're just creating noise due to the bundler 2.x issues